test/beacon_web/live/page_live_test.exs

```diff
@@ -39,21 +39,40 @@ defmodule BeaconWeb.Live.PageLiveTest do
         """
       })
 
-    Pages.create_page!(%{
-      path: "home",
-      site: "my_site",
-      layout_id: layout.id,
-      template: """
-      <main>
-        <h2>Some Values:</h2>
-        <ul>
-          <%= for val <- @beacon_live_data[:vals] do %>
-            <%= my_component("sample_component", val: val) %>
-          <% end %>
-        </ul>
-      </main>
+    page =
+      Pages.create_page!(%{
+        path: "home",
+        site: "my_site",
+        layout_id: layout.id,
+        template: """
+        <main>
+          <h2>Some Values:</h2>
+          <ul>
+            <%= for val <- @beacon_live_data[:vals] do %>
+              <%= my_component("sample_component", val: val) %>
+            <% end %>
+          </ul>
+
+          <.form let={f} for={:greeting} phx-submit="hello">
+            Name: <%= text_input f, :name %>
+            <%= submit "Hello" %>
+          </.form>
+
+          <%= if assigns[:message], do: assigns.message %>
+        </main>
+        """
+      })
+
+    Pages.create_page_event!(%{
+      page_id: page.id,
+      event_name: "hello",
+      code: """
+        {:noreply, Phoenix.LiveView.assign(socket, :message, "Hello \#{event_params["greeting"]["name"]}!")}
       """
     })
+
+    # Make sure events are loaded.
+    Beacon.Loader.DBLoader.load_from_db()
```

Comment:
I wonder if we should also reload it after inserting the Event as we do for Pages 🤔